app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -145,7 +148,14 @@ public static String localizeViewCount(Context context, long viewCount) {
     }
 
     public static String localizeStreamCount(Context context, long streamCount) {
-        return getQuantity(context, R.plurals.videos, R.string.no_videos, streamCount, localizeNumber(context, streamCount));
+        switch ((int) streamCount) {
```

Comment:
Why are you casting to `int`?

Reply:
Because Java `switch` does not allow the usage of `long`

Reply:
Then just use an `if`, as this would cause issues when the `streamCount` Is e.g. 4294967294.

Reply:
Yeah but I don't think a stream count can reach 2^31 - 1 or  2147483647 or 2*10^10, that's really big. Also, it would be pointless to prevent issues here, if `streamCount` has to be cast to `int` anyway in order to pass it to `getQuantity`.

Reply:
I didn't see `getQuantity` needed an `int`, so in that case, that's fine.